pages/web_cloud/internet/overthebox/offer_migration/guide.fr-fr.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Comment changer mon offre OverTheBox
+updated: 2024-07-17
+---
+
+## Objectif
+
+Vous pouvez faire évoluer votre offre OverTheBox afin de pouvoir accéder à de nouvelles fonctionnalités et services.
+
+Le changement d'offre permet de :
+
+* Migrer d'une offre Starter 500Mbps à une offre Business 1Gbps.
+* Migrer d'une offre Business 1Gbps à une offre Starter 500 Mbps.
+* Migrer d'une ancienne offre de service à une nouvelle offre Starter ou Business.
+* Commander un boitier OTBv3
```

Comment:
Boitier as of 1990 orthography rules should we revert to old orthography ?
https://www.dictionnaire-academie.fr/article/A9B1480